cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1317,11 +1317,43 @@ declare namespace Deno {
    */
   export function openKv(path?: string): Promise<Deno.Kv>;
 
+  export interface Schedule {
```

Comment:
Please name this `CronSchedule` (consistent with other type/interface names)

Reply:
Done.